src/main/webapp/scripts/map.js

```diff
@@ -12,6 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* Custom Markers */
+const yellowIcon = {
+  url: "/images/yellow-marker.png", // url
+  scaledSize: new google.maps.Size(30, 40), // scaled size
+  origin: new google.maps.Point(0, 0), // origin
+  anchor: new google.maps.Point(0, 0) // anchor
+};
+
+const blueIcon = {
+  url: "/images/blue-marker.png", // url
+  scaledSize: new google.maps.Size(30, 40), // scaled size
+  origin: new google.maps.Point(0, 0), // origin
+  anchor: new google.maps.Point(0, 0) // anchor
```

Comment:
can we avoid duplicating the same values used for all the icons? maybe by having a generic icon holding these values and only override the url for each colour?

Reply:
yes I think so - there's currently a few issues with where to put the markers, but I'll reduce the code repetition in the next commit.